sycl/test-e2e/Matrix/element_wise_all_ops_int8_packed_impl.hpp

```diff
@@ -185,31 +169,28 @@ void matrix_verify_logic(queue q, big_matrix<T, M, N> &A, nd_range<2> &r,
 
            sub_group sg = spmd_item.get_sub_group();
            joint_matrix<sub_group, int8_t, use::b, TK, TN,
-                        ext::intel::experimental::matrix::layout::packed>
+                        layout::ext_intel_packed>
                sub_b;
 
            joint_matrix_fill(sg, sub_b, 5);
 
-           auto wi_slice_b =
-               sycl::ext::intel::experimental::matrix::get_wi_data(sg, sub_b);
-           for (int i = 0; i < wi_slice_b.length(); i++) {
-             if (wi_slice_b[i]) {
-               if (wi_slice_b[i] > 2 || wi_slice_b[i] >= 2 ||
-                   wi_slice_b[i] < 2 || wi_slice_b[i] <= 2) {
-                 T val = (wi_slice_b[i] != 2) ? wi_slice_b[i] : 2;
+           joint_matrix_apply(sg, sub_b, [](T &x) {
```

Comment:
Same as above. In my opinion, test can be reduced to one joint_matrix_apply call.